dekf_sensor_fusion/globalCovariance poscov
---
bool success
float64 result
